clang/test/dpct/cuda-device-api.cu

```diff
@@ -35,25 +35,28 @@ void foo() {
   // CHECK-NEXT: */
   cudaSetDeviceFlags(flags);
 
-  // CHECK: /*
+  // CHECK:   /*
   // CHECK-NEXT: DPCT1026:{{[0-9]+}}: The call to cudaDeviceEnablePeerAccess was removed because SYCL
-  // CHECK-NEXT: currently does not support memory access across peer devices.
+  // CHECK-NEXT: currently does not support memory access across peer devices. You can specify
+  // CHECK-NEXT: "--use-dpcpp-extensions=peer_access" to use the extensions to migrate
+  // CHECK-NEXT: cudaDeviceEnablePeerAccess.
   // CHECK-NEXT: */
   cudaDeviceEnablePeerAccess(peerDevice, flags);
 
   // CHECK: /*
   // CHECK-NEXT: DPCT1026:{{[0-9]+}}: The call to cudaDeviceDisablePeerAccess was removed because SYCL
-  // CHECK-NEXT: currently does not support memory access across peer devices.
+  // CHECK-NEXT: currently does not support memory access across peer devices. You can specify
+  // CHECK-NEXT: "--use-dpcpp-extensions=peer_access" to use the extensions to migrate
+  // CHECK-NEXT: cudaDeviceDisablePeerAccess.
   // CHECK-NEXT: */
   cudaDeviceDisablePeerAccess(peerDevice);
 
-  // CHECK:      /*
-  // CHECK-NEXT: DPCT1031:{{[0-9]+}}: Memory accessing across peer devices is an implementation-specific
-  // CHECK-NEXT: feature which may not be supported by some SYCL backends and compilers. The
-  // CHECK-NEXT: output parameter(s) are set to 0.
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1026:{{[0-9]+}}: The call to cudaDeviceCanAccessPeer was removed because SYCL
```

Comment:
let's keep use DOCT1031, do the migration action: replace the API with a assign expr.  At same time ask user : You can migrate the code with peer access extension by not specifying -no-use-dpcpp-extensions=peer_access.